.husky/_/husky.sh

```diff
@@ -1,10 +1,12 @@
-#!/bin/sh
+#!/usr/bin/env sh
```

Comment:
it seems husky installed incorrectly 
- should be pre_commit file instead
- should be husky install at package.json "post install" script
please check my PR https://github.com/GoodDollar/GoodWeb3-Mono/pull/31 and husky docs https://typicode.github.io/husky/#/?id=install-1